gnovm/tests/files/addressable_7b_err.gno

```diff
@@ -0,0 +1,9 @@
+package main
+
+type MyInt int
+
+func main() {
+	_ = &MyInt
+}
+
+// Error:
```

Comment:
No error msg?

Reply:
Ha, yeah you're right.  I removed this test. It should fail, but doesn't. I created an issue for this here: https://github.com/gnolang/gno/issues/2787